Gopkg.toml

```diff
@@ -91,7 +91,7 @@ required = [
 
 [[constraint]]
   name = "github.com/cloudevents/sdk-go"
-  version = "1.0.0"
+  revision = "427e8cf75237fdc7d4bb6c496398d7288efb8921"
```

Comment:
not sure... we want to use a revision here - instead of a _released_ version ?
/cc @n3wscott

Reply:
We can wait for a minor version release, I just noticed we've used revisions for other packages.

Reply:
This is using cutting edge stuff. It is ok